pkg/segment/structs/evaluationstructs.go

```diff
@@ -1380,7 +1396,7 @@ func (self *StatisticExpr) SortBucketResult(results *[]*BucketResult) error {
 
 // Only display fields which in StatisticExpr
 func (self *StatisticExpr) RemoveFieldsNotInExprForBucketRes(bucketResult *BucketResult) error {
-	groupByCols := self.GetGroupByCols()
+	groupByCols := self.GetFields()
```

Comment:
hmm... could we come up with a better name for this func ? It is getting the column names but then assign it to a "group by column name field"

Reply:
It is getting `byClause` and `FieldList`. This is the definition of this function.

```
func (self *StatisticExpr) GetGroupByCols() []string {
	return append(self.FieldList, self.ByClause...)
}
```